src/components/common/Button/index.tsx

```diff
@@ -1,8 +1,8 @@
 import React from 'react'
-import styled, { css, ThemeProvider } from 'styled-components'
-import { ThemeMap, ThemeValue, variants } from 'styled-theming'
+import styled, { css, /* DefaultTheme, */ ThemeProvider } from 'styled-components'
```

Comment:
```suggestion
import styled, { css, ThemeProvider } from 'styled-components'
```
?

Reply:
i use it later in the chain so I will ignore this for now but thanks!